cherab/jet/spectroscopy/kt3/kt3.py

```diff
@@ -0,0 +1,405 @@
+import numpy as np
+from math import sqrt, tan, radians
+
+from jet.data import sal
```

Comment:
I think the version of sal from jet package is slightly different isn't it? At least according to what they say [here](https://data.jet.uk/guides/sal/python_client.html#show-viewing-data). It seems like this is the SAL version to use for JET. Also, when the default server changes, it should automatically be changed in the jet package, shouldn't it? This way we wouldn't have to care. The same style is used also in the equilibrium loading. Am I missing something?

Reply:
You are correct. My point is that with `from jet.data import sal` this code will run only at JET clusters. With `from sal.client import SALClient` you can run it from anywhere.

Reply:
Look like `sal = SALClient('https://sal.jet.uk')` requires authorization on JET Data Center nodes, while `from jet.data import sal` works without authorization because it's configured for https://sal.jetdata.eu, which is currently unavailable outside of the JDC. 
Something like:
```python
try:
    from jet.data import sal
except ModuleNotFoundError:
    # use general SAL client outside of the JET computing infrastructure
    from sal.client import SALClient
    sal = SALClient('https://sal.jet.uk')
```
will be a better solution. 

However, this problem is general for all JET diagnostics implemented in Cherab, and should probably be solved with 
```python
from cherab.jet import sal
```
So, please ignore my comments about this issue, I think it's OK to use `from jet.data import sal` for now.

Reply:
@Mateasek pointed out that the JET version of the SAL client can be installed on local machines by following the instructions from https://data.jet.uk/guides/sal/downloads.html#sal-downloads-label. Thus, replacing `jet.data.sal` with a generic `sal` does not make `cherab.jet` more portable as I mistakenly thought.